docs-ref-services/latest/resourcemanager-iothub-readme.md

```diff
@@ -108,7 +107,7 @@ For details on contributing to this repository, see the [contributing guide](htt
 <!-- LINKS -->
 [survey]: https://microsoft.qualtrics.com/jfe/form/SV_ehN0lIk2FKEBkwd?Q_CHL=DOCS
 [docs]: https://azure.github.io/azure-sdk-for-java/
-[jdk]: https://docs.microsoft.com/java/azure/jdk/
+[jdk]: /java/azure/jdk/
```

Comment:
This part of the file is automated from readme in SDK repo. It cannot be changed to relative path in SDK repo since relative path is not allowed and there is no msdoc on that relative path anyway.